tests/test_assert/test_assert.c

```diff
@@ -0,0 +1,34 @@
+/* 
```

Comment:
Rename to test_abort or similar and just call solo5_abort() directly. (See comments above).